src/css/_time-span-pickers.scss

```diff
@@ -24,6 +24,7 @@
 
   input {
     @include label-small;
+    @include input-focus;
```

Comment:
Makes editing the time span look like this: 
<img width="392" alt="Screen Shot 2020-06-22 at 12 33 16 PM" src="https://user-images.githubusercontent.com/3460638/85329708-e0017d00-b487-11ea-8d78-31cc45daa89e.png">